team-builder/src/Form.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+
+const Form = (props) => {
+  const [form, setForm] = useState({
+    name: "",
+    email: "",
+    role: "",
+  });
+
+  const changeHandler = (e) => {
+    setForm({ ...form, [e.target.name]: e.target.value });
+  };
+
+  const submitForm = (e) => {
+    e.preventDefault();
+    props.addNewPerson(form);
+    setForm({ ...form, ["name"]: "", ["email"]: "", ["role"]: "" });
+  };
+  return (
+    <div>
+      <form>
```

Comment:
prior to sumbitting this review, I would like to review this `line 21` and `line 52` to see how we can achieve a greater amount of `acceseibility` to this form.